
def a():
    raise Exception('some exc')
